src/scheduler/pct.rs

```diff
@@ -42,6 +42,21 @@ impl PctScheduler {
     pub fn new_from_seed(seed: u64, max_depth: usize, max_iterations: usize) -> Self {
```

Comment:
Need to update the comment to explain the effect of setting `SHUTTLE_RANDOM_SEED`